src/test/resources/application.yml

```diff
@@ -1,14 +1,19 @@
 server:
-  port: 8080
+  port: 8081
 endpointUrl:
-  retrieve: http://localhost:${server.port}
+  store: http://localhost:9041/mis/product/
+  retrieve: http://localhost:${server.port}/
+  transform: http://localhost:9090/transform/
```

Comment:
The slash at the end of these url strings should not be needed

Reply:
at least at one point, the retrieve endpoint was responding without it so we would see results like `http://localhost:808005b04fc7bd0a497b9e946a75aeec293b` Haven't investigated yet